README.md

```diff
@@ -1 +1,4 @@
-# service-yolo-vino
\ No newline at end of file
+# service-yolo-vino
+![](https://img.shields.io/github/license/julianchen215/service-yolo-vino)
```

Comment:
Please change the license shield path from "julianchen215" to "open-horizon-services"